### PR TITLE
Document how to run manual CI builds.

### PR DESCRIPTION
**Pull request by david-german-tri**
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/2496)

### CODE REVIEW

File: drake/doc/jenkins.rst [hunk 1]
```diff
@@ -12,4 +15,26 @@ Respond:
...
+You can also view the `Jenkins UI <https://drake-jenkins.csail.mit.edu/>`_
+directly.
+
+Running a Specific Build Before Submitting
+==========================================
+
+There are a number of Jenkins builds that do not normally run pre-submit, but
+do run post-submit or nightly.  The post-submit and nightly builds include
+long-running tests, lower-priority platforms (e.g. Windows MATLAB), and
+specialized options (e.g. msan).  Users with login access to Jenkins can
```
Comment (jamiesnape) [inline drake/doc/jenkins.rst]: (members of the RobotLocomotion organization have this access by default)

**Comment by david-german-tri** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[drake/doc/jenkins.rst, line 27 \[r1\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJCYoggTt8a945Vy9G9-r1-27:-KJC_HgkSzCLeizKJ0Zv:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/174a9105a9810eb9d288e93c4755be5884a27c8e/drake/doc/jenkins.rst#L27)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> (members of the RobotLocomotion organization have this access by default)
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by liangfok** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---

*[drake/doc/jenkins.rst, line 21 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJDB9yCmSW3ZTZoA_ib:-KJDB9yCmSW3ZTZoA_ic:1322337833) ([raw file](https://github.com/robotlocomotion/drake/blob/e1fd15e5100e6f351b2d2055004a33f178550e92/drake/doc/jenkins.rst#L21)):*
> ```reStructuredText
> directly.
> 
> Running a Specific Build Before Submitting
> ```

BTW I think it's best practice to explicitly define an anchor above the section title:

```
.. _run_specific_build:
```

This would allow the title to change in the future without the anchor changing.

---

*[drake/doc/jenkins.rst, line 25 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJDAizQftWZZRRtuX2c:-KJDAizRDSuUVCEnblpb:1247932366) ([raw file](https://github.com/robotlocomotion/drake/blob/e1fd15e5100e6f351b2d2055004a33f178550e92/drake/doc/jenkins.rst#L25)):*
> ```reStructuredText
> 
> There are a number of Jenkins builds that do not normally run pre-submit, but
> do run post-submit or nightly.  The post-submit and nightly builds include
> ```

I'm a little confused by the terminology "pre-submit" and "post-submit" since I believe "submit" refers to the _creation_ of the pull request. Wouldn't "pre-merge" and "post-merge" be better terminology since it clearly means "CI tests that are run _after_ a pull request is merged"?

---

*[drake/doc/jenkins.rst, line 34 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJDBo41T_2umtQM9aci:-KJDBo41T_2umtQM9acj:2101266382) ([raw file](https://github.com/robotlocomotion/drake/blob/e1fd15e5100e6f351b2d2055004a33f178550e92/drake/doc/jenkins.rst#L34)):*
> ```reStructuredText
> 2. Go to the `list of Experimental builds <https://drake-jenkins.csail.mit.edu/view/Experimental/>`_.
> 
>    a. In this list, the build named "experimental" runs on presubmit.
> ```

Can the builds that run on post submit be easily identified?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by david-german-tri** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[drake/doc/jenkins.rst, line 21 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJDB9yCmSW3ZTZoA_ib:-KJDDlrcFWQt2oh99N3o:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/e1fd15e5100e6f351b2d2055004a33f178550e92/drake/doc/jenkins.rst#L21)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> BTW I think it's best practice to explicitly define an anchor above the section title:
> 
> ```
> .. _run_specific_build:
> ```
> 
> This would allow the title to change in the future without the anchor changing.
</details>
Done.

---

*[drake/doc/jenkins.rst, line 25 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJDAizQftWZZRRtuX2c:-KJDF56akt3AFBrSZoDc:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/e1fd15e5100e6f351b2d2055004a33f178550e92/drake/doc/jenkins.rst#L25)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> I'm a little confused by the terminology "pre-submit" and "post-submit" since I believe "submit" refers to the _creation_ of the pull request. Wouldn't "pre-merge" and "post-merge" be better terminology since it clearly means "CI tests that are run _after_ a pull request is merged"?
</details>
Done.

---

*[drake/doc/jenkins.rst, line 34 \[r2\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJDBo41T_2umtQM9aci:-KJDF6v4Dus6V3qTdz5X:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/e1fd15e5100e6f351b2d2055004a33f178550e92/drake/doc/jenkins.rst#L34)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Can the builds that run on post submit be easily identified?
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by liangfok** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[drake/doc/jenkins.rst, line 47 \[r3\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJDGSDetZQpqbdq5BMY:-KJDGSDetZQpqbdq5BMZ:193045614) ([raw file](https://github.com/robotlocomotion/drake/blob/8c02727a1dab5439d02712a224487ca3b0835557/drake/doc/jenkins.rst#L47)):*
> ```reStructuredText
> and the
> `list of Nightly builds <https://drake-jenkins.csail.mit.edu/view/Nightly/>`_,
> but you can't trigger those builds directly.
> ```

I see. So the nightly build that typically identifies the need for a revert cannot be directly run. However, builds with similar configurations are available in the list of Continuous builds, which _can_ be directly run?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by liangfok** [comment]
Just one more question.

<details><summary><i>Previously, david-german-tri (David German) wrote…</i></summary>
> #### Document how to run manual CI builds\.
</details>


---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-:-KJDHHMTMs6JUfiAgR3j:1280101350)*

**Comment by david-german-tri** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[drake/doc/jenkins.rst, line 47 \[r3\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJDGSDetZQpqbdq5BMY:-KJGG66AEa1waHl4fFXa:2020083710) ([raw file](https://github.com/robotlocomotion/drake/blob/8c02727a1dab5439d02712a224487ca3b0835557/drake/doc/jenkins.rst#L47)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> I see. So the nightly build that typically identifies the need for a revert cannot be directly run. However, builds with similar configurations are available in the list of Continuous builds, which _can_ be directly run?
</details>
Clarified the text.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by liangfok** [comment]
:lgtm_strong: 

<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> Just one more question.
</details>


---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-:-KJGGvkFvoc0bWbEBTaJ:-1868698381)*

**Comment by david-german-tri** [comment]
+@jwnimmer-tri for platform review

<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> :lgtm_strong: 
</details>


---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-:-KJGVrmufsmh70LhlL0a:-58081829)*

**Comment by liangfok** [comment]
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---

*[drake/doc/jenkins.rst, line 29 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJGZqKob8KPQCKHMAig:-KJGZqKob8KPQCKHMAih:1710136730) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L29)):*
> ```reStructuredText
> do run post-merge or nightly.  The post-merge and nightly builds include
> long-running tests, lower-priority platforms (e.g. Windows MATLAB), and
> specialized options (e.g. msan). Members of the RobotLocomotion organization
> ```

BTW, consider making "msan" a link to provide more semantic context:

https://github.com/google/sanitizers/wiki/MemorySanitizer

---

*[drake/doc/jenkins.rst, line 31 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJGZzMpiizllajjhxLx:-KJGZzMpiizllajjhxLy:727804880) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L31)):*
> ```reStructuredText
> specialized options (e.g. msan). Members of the RobotLocomotion organization
> can manually trigger these builds on pull requests that have not yet been
> merged.
> ```

BTW, perhaps it would be clarified that the pull request can originate from anywhere including forks and need not be from a branch within `RobotLocomotion/drake`.

---

*[drake/doc/jenkins.rst, line 33 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJG_DZXNsO7nRUTRDK4:-KJG_DZXNsO7nRUTRDK5:-901010900) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L33)):*
> ```reStructuredText
> merged.
> 
> 1. Log in to Jenkins using GitHub OAuth.
> ```

BTW, consider making "Jenkins" a link to https://drake-jenkins.csail.mit.edu/.

---

*[drake/doc/jenkins.rst, line 42 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJG_Q0nG_4R_0-lwWwh:-KJG_Q0nG_4R_0-lwWwi:1034661567) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L42)):*
> ```reStructuredText
> 
> The list of Experimental builds includes a build named ``experimental``, which
> runs pre-merge, and numerous other builds for one-off use. To help identify the
> ```

BTW, consider changing "one-off" to be "manually-triggered"

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by jamiesnape** [comment]
Review status: all files reviewed at latest revision, all discussions resolved.

---

*[drake/doc/jenkins.rst, line 31 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJGZzMpiizllajjhxLx:-KJG_srvKMJ1MaDDQBFl:1112045240) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L31)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> BTW, perhaps it would be clarified that the pull request can originate from anywhere including forks and need not be from a branch within `RobotLocomotion/drake`.
</details>
To be precise, only a pull request may be from a fork. However, revert PRs are by default created as a branch on RobotLocomotion/drake, so they may be tested by branch name, if desired.

---

*[drake/doc/jenkins.rst, line 42 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJG_Q0nG_4R_0-lwWwh:-KJGaPGagZhC7Ngjnd0n:-965541159) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L42)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> BTW, consider changing "one-off" to be "manually-triggered"
</details>
"On-demand" is the terminology currently used on Jenkins. "Scheduled" is also more correct than trigger as Jenkins always sends jobs first to a queue. These jobs have high priority, but do not jump the queue entirely.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by jwnimmer-tri** [comment]
:lgtm:

<details><summary><i>Previously, david-german-tri (David German) wrote…</i></summary>
> +@jwnimmer-tri for platform review
</details>


---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-:-KJGbZlM2typEVCGHrQ5:1417088721)*

**Comment by david-german-tri** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[drake/doc/jenkins.rst, line 29 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJGZqKob8KPQCKHMAig:-KJGcNnDv4rlpj7pNuSx:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L29)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> BTW, consider making "msan" a link to provide more semantic context:
> 
> https://github.com/google/sanitizers/wiki/MemorySanitizer
</details>
Done.

---

*[drake/doc/jenkins.rst, line 31 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJGZzMpiizllajjhxLx:-KJGd0-ufqeokeOfQrhZ:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L31)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> To be precise, only a pull request may be from a fork. However, revert PRs are by default created as a branch on RobotLocomotion/drake, so they may be tested by branch name, if desired.
</details>
Done.

---

*[drake/doc/jenkins.rst, line 33 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJG_DZXNsO7nRUTRDK4:-KJGcWP5a29S_ZZCV4ua:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L33)):*
<details><summary><i>Previously, liangfok (Chien-Liang Fok) wrote…</i></summary>
> BTW, consider making "Jenkins" a link to https://drake-jenkins.csail.mit.edu/.
</details>
Done.

---

*[drake/doc/jenkins.rst, line 42 \[r4\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-KJG_Q0nG_4R_0-lwWwh:-KJGcMOTB58RAXd3toJG:-499146009) ([raw file](https://github.com/robotlocomotion/drake/blob/f7621a0e6b0281c046d166d20fbe0e274caabea7/drake/doc/jenkins.rst#L42)):*
<details><summary><i>Previously, jamiesnape (Jamie Snape) wrote…</i></summary>
> "On-demand" is the terminology currently used on Jenkins. "Scheduled" is also more correct than trigger as Jenkins always sends jobs first to a queue. These jobs have high priority, but do not jump the queue entirely.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by jwnimmer-tri** [comment]
Reviewed 1 of 1 files at r5, 1 of 1 files at r6.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by liangfok** [comment]
:lgtm_strong: 

<details><summary><i>Previously, jwnimmer-tri (Jeremy Nimmer) wrote…</i></summary>
> :lgtm:
</details>


---

Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496#-:-KJGvNa6BOpZqyhAa2Wn:-1868698381)*

**Comment by david-german-tri** [comment]
jenkins: retest this please

**Comment by liangfok** [comment]
Reviewed 1 of 1 files at r6.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2496)*

**Comment by jwnimmer-tri** [comment]
Using override-merge due to out-of-date-with-master.

**Pull request merged**